app/models/sensor.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: sensors
+#
+#  id          :bigint           not null, primary key
+#  device_id   :bigint
+#  description :string
+#
+# Indexes
+#
+#  index_sensors_on_device_id  (device_id)
+#
+class Sensor < ApplicationRecord
```

Comment:
May be some more information like location? type? Or something to identify which sensor in the room is

Reply:
Maybe we can add the location to the device instead of the sensor.
Also, we have a description so maybe we can use it for the name of the room or something like that